helm/default-apps-aws/values.yaml

```diff
@@ -204,3 +204,15 @@ apps:
     # used by renovate
     # repo: giantswarm/vertical-pod-autoscaler-crd
     version: 1.0.1
+  observability-bundle:
+    appName: observability-bundle
+    chartName: observability-bundle
+    catalog: default-test
+    namespace: ""
+    inCluster: true
+    clusterValues:
+      configMap: true
+      secret: false
+    # used by renovate
+    # repo: giantswarm/observability-bundle
+    version: 0.1.1-a15644eb8bc8362ddad27b1a69ee15c651fd9db6
```

Comment:
Please make sure this is correct before merging

Reply:
Of course!

Reply:
👍  No worries :) Just wanted to make sure as Jose approved.